core/appeal/service.go

```diff
@@ -8,9 +8,9 @@ import (
 
 	"github.com/go-playground/validator/v10"
 	"github.com/odpf/guardian/domain"
-	"github.com/odpf/guardian/internal/evaluator"
+	"github.com/odpf/guardian/internal/store"
+	"github.com/odpf/guardian/pkg/evaluator"
 	"github.com/odpf/guardian/plugins/notifiers"
-	"github.com/odpf/guardian/store"
 	"github.com/odpf/guardian/utils"
 	"github.com/odpf/salt/log"
 )
```

Comment:
If it is possible, better to move repository interface here and we can create a service interface inside server

Reply:
ah i got it. yes, part of it already implemented on another pr here https://github.com/odpf/guardian/pull/155/files#diff-4ceaf11357ea8ac162487cfbb8f9f0a0a7446fa7b61bc799aac1a3d3ca7adc58R39-R41, even though it's still look like this
```go
type repository interface {
	store.AppealRepository
}
```
will write the methods directly instead of embedding `store.AppealRepository`

Reply:
@mabdh doing this one from the other branch to avoid major conflict https://github.com/odpf/guardian/pull/155/commits/7a96ae36d594733364ce27d6204bc96111a04c51 🙏